generator/integration_tests/tests/golden_kitchen_sink_stub_factory_test.cc

```diff
@@ -68,6 +70,45 @@ TEST_F(GoldenKitchenSinkStubFactoryTest, DefaultStubWithAuth) {
   EXPECT_THAT(context.credentials(), Not(IsNull()));
 }
 
+TEST_F(GoldenKitchenSinkStubFactoryTest, DefaultStubWithUniverseDomainOption) {
+  Options options;
+  options.set<EndpointOption>("localhost:1")
+      .set<internal::UniverseDomainOption>("not empty")
+      .set<UnifiedCredentialsOption>(MakeAccessTokenCredentials(
+          "invalid-access-token",
+          std::chrono::system_clock::now() + std::chrono::minutes(15)));
+  auto default_stub =
+      CreateDefaultGoldenKitchenSinkStub(CompletionQueue{}, options);
+  grpc::ClientContext context;
+  auto response =
+      default_stub->GenerateIdToken(context, GenerateIdTokenRequest{});
+  EXPECT_THAT(response, Not(IsOk()));
+  EXPECT_THAT(
+      response,
+      Not(StatusIs(StatusCode::kInvalidArgument,
```

Comment:
Nit:  It is a little strange to expect that a status is not a particular variety.  Is there a positive way to express the expectation?